src/python/grpcio_tests/tests/qps/README.md

```diff
@@ -0,0 +1,100 @@
+# Python Benchmark Tools
+
+## Scenarios
+
+In `src/proto/grpc/testing/control.proto`, it defines the fields of a scenario.
+In `tools/run_tests/performance/scenario_config.py`, the script generates actual scenario content that usually in json format, or piped to another script.
+
+All Python related benchmark scenarios are:
+* netperf
+* python_generic_sync_streaming_ping_pong
+* python_protobuf_sync_streaming_ping_pong
+* python_protobuf_async_unary_ping_pong
+* python_protobuf_sync_unary_ping_pong
+* python_protobuf_sync_unary_qps_unconstrained
+* python_protobuf_sync_streaming_qps_unconstrained
+* python_protobuf_sync_unary_ping_pong_1MB
+
+Here I picked the top 2 most representative scenarios of them, and reduce their benchmark duration from 30 seconds to 10 seconds:
```

Comment:
Is "most representative" backed by an objective measurement? If not its fine, perhaps just reword to "I [we? I think we usually use we in the readme files] picked a small but representative subset"

Reply:
Updated.